packages/unified-latex-util-pegjs/grammars/latex.pegjs

```diff
@@ -286,6 +291,11 @@ macro "macro"
             return createNode("macro", { content: m });
         }
 
+text_macro "text macro"
```

Comment:
I don't like the idea of adding specific macros to the grammar unless absolutely necessary. `\text` is one of the ways to get text inside of math. But there are others, and a user could easily define custom ones.

Reply:
Agree, but we still need entry point to text mode (like `$$ `is entry point to math mode)

Reply:
I think the solution is to reparse the macro contents after the initial parse.

Reply:
It forces us to copy a banch of "text-logic" from peg grammar to reparse method.

Reply:
I think you're seeing why I said it was such a hard problem...

Right now, macros can set `renderInfo: { inMathMode: true }`. I think we can set `renderInfo: { inMathMode: false }` for macros like `\text`. If in parsing we encounter one of these, then we would grab the original string and reparse. This could make parsing with a lot of nested math/text wasteful, but I don't think people do that so much in practice.

Reply:
Can you reverse the commit where you added a `text_macro` to the grammar?

Reply:
Done